src/algorithm.rs

```diff
@@ -178,6 +178,10 @@ where
     pub(crate) fn process(&mut self, msg: BftMsg, need_wal: bool) -> BftResult<()> {
         match msg {
             BftMsg::Proposal(encode) => {
+                info!(
```

Comment:
is this a necessary output info for production runtime? if not, should use debug level logging.

Reply:
You are right.

I think over about this code line, it should be in trace level. 

Fixed now.